### DISCUSSION TITLE
"add key at cursor" button in timeline controls doesn't seem to work

### DISCUSSION THREAD

**Discussion by mwozniczak** [cables-gl/cables_docs#245] [! Welcome - readme first!]
when attempting to use the "add key at cursor" button to (presumably?) add a keyframe, I get this error instead:

![image](https://user-images.githubusercontent.com/14187903/169895045-6a14f6c6-87db-4ebc-8ebc-6dfa235e76ad.png)

All the steps necessary to reproduce this (in my case, at least) is clicking on the plus button here:

![image](https://user-images.githubusercontent.com/14187903/169895265-0200fe0e-d9d0-49da-92d4-3540ffa12be8.png)

OS: Manjaro Linux
Browsers tried: Firefox, Chrome, MS Edge (all current stable versions)

**Comment by franckaubry** [comment]
Can you do screen recording or something similar so that we can see all the ops/selections/steps you are doing to get this error?

I cannot reproduce your error either. Tried on similar OS and browsers.

**Reply by steam0r** [reply]
can you also try this on dev.cables.gl and see if it happens there as well? thanks!

**Reply by mwozniczak** [reply]
okay so i've checked the dev version and it does not occur there

on the other hand, you click on it and there's no indication whether or not it did anything (which, since i didn't have anything animated selected, it doesn't)

**Reply by steam0r** [reply]
good to hear at least that crash is gone, maybe there indeed should be some notice when it can't do anything with nothing set to animated, we will keep this in mind when working on the timeline. thanks for reporting and testing!

**Comment by mwozniczak** [comment]
i _think_  to reproduce this you need to not have anything that's set to be animated in your project yet. so, start a new project, press that button.

now granted, i do understand that the purpose of this is to add keyframes to something that already exists, but still:

1. i don't think the raw error helps the end user and
2. that _could_ be a good opportunity to teach the user how to _actually_ use keyframes and the whole timeline?

but yeah, i'll have a look if it's any different on dev
